app/models/validations/sales/sale_information_validations.rb

```diff
@@ -108,20 +108,20 @@ def validate_discount_and_value(record)
 
   def validate_non_staircasing_mortgage(record)
     return unless record.value && record.deposit && record.equity
-    return unless record.is_not_staircasing?
+    return unless record.is_not_staircasing? && record.type != 18
     return unless record.saledate && record.form.start_year_after_2024?
 
     if record.mortgage_used?
       return unless record.mortgage
 
       if record.mortgage_and_deposit_total != record.expected_shared_ownership_deposit_value
-        %i[mortgage value deposit equity].each do |field|
+        %i[mortgage value deposit equity type].each do |field|
```

Comment:
do we need to change the validation message to make it clear this is type-dependent now?